Exercise_C/index.html

```diff
@@ -1,16 +1,27 @@
-<!DOCTYPE html>
-<html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-  <body>
-    <!-- CODE HERE -->
-
-    <script src="" async defer></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title></title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <!-- CODE HERE -->
+    <div class="flex_container-1">
+      <div class="blue Box" style="align-self: flex-start"></div>
+    
+    
+      <div class="green Box" style="align-self: flex-end"></div>
+  </div>
+  <div class="flex_container-2">    
+    <div class="red Box">
+    </div>
+  </div>
+
+    <script src="" async defer></script>
+  </body>
+</html>
+

```

Comment:
Hello Nicole, 
Great work on this exercise and making use of Flex to position the items